lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -546,6 +546,17 @@ module.exports = class MethodMappings {
 
           return elementText;
         },
+        async isElementPresent(id) {
+          const element = this.getWebElement(id);
+        
+          try {
+            await element.findElement();
+            return true;
+          } catch (error) {
+            return false;
+          }
+        }
```

Comment:
This won't work. Did you try to run the `.isVisible()` command yourself in one of the Nightwatch example tests and seeing true/false getting returned?

Reply:
async isElementPresent(id) {
          try {
              const element = await this.getWebElement(id);
              return true; // Element found, so it's present
          } catch (error) {
              if (error.name === 'NoSuchElementError') {
                  return false; // Element not found, so it's not present
              } else {
                  throw error; // Rethrow any other error
              }
          }
      }  
i have tried this i am able to catch the error in the terminal but not in report file 
![Screenshot 2024-03-02 011955](https://github.com/nightwatchjs/nightwatch/assets/117269123/c1a35179-2e4b-48c5-8cbd-7d251b9c1a24)